Build fails with `-Werror`: `qjsc.c: error: '%d' directive output may be truncated writing between 1 and 10 bytes into a region of size between 0 and 1023`

```
[ 68%] Building C object CMakeFiles/qjs_exe.dir/qjs.c.o
/opt/local/bin/gcc-mp-14 -D_GNU_SOURCE -Dqjs_exe_EXPORTS -I/opt/local/var/macports/build/_opt_local_ppcports_devel_quickjs-ng/quickjs-ng/work/quickjs-0.9.0 -pipe -Os -DNDEBUG -isystem/opt/local/include/LegacySupport -I/opt/local/include -std=gnu11 -arch ppc -mmacosx-version-min=10.6 -fvisibility=hidden -Wall -Werror -Wextra -Wformat=2 -Wno-implicit-fallthrough -Wno-sign-compare -Wno-missing-field-initializers -Wno-unused-parameter -Wno-unused-but-set-variable -Wno-unused-result -Wno-stringop-truncation -Wno-array-bounds -funsigned-char -Wno-maybe-uninitialized -MD -MT CMakeFiles/qjs_exe.dir/qjs.c.o -MF CMakeFiles/qjs_exe.dir/qjs.c.o.d -o CMakeFiles/qjs_exe.dir/qjs.c.o -c /opt/local/var/macports/build/_opt_local_ppcports_devel_quickjs-ng/quickjs-ng/work/quickjs-0.9.0/qjs.c
In file included from /usr/include/stdio.h:443,
                 from /opt/local/include/LegacySupport/stdio.h:38,
                 from /opt/local/var/macports/build/_opt_local_ppcports_devel_quickjs-ng/quickjs-ng/work/quickjs-0.9.0/qjsc.c:27:
In function 'find_unique_cname',
    inlined from 'jsc_module_loader.part.0' at /opt/local/var/macports/build/_opt_local_ppcports_devel_quickjs-ng/quickjs-ng/work/quickjs-0.9.0/qjsc.c:262:13:
/opt/local/var/macports/build/_opt_local_ppcports_devel_quickjs-ng/quickjs-ng/work/quickjs-0.9.0/qjsc.c:216:42: error: '%d' directive output may be truncated writing between 1 and 10 bytes into a region of size between 0 and 1023 [-Werror=format-truncation=]
  216 |         snprintf(cname1, sizeof(cname1), "%s_%d", cname, suffix_num);
      |                                          ^~~~~~~
/opt/local/var/macports/build/_opt_local_ppcports_devel_quickjs-ng/quickjs-ng/work/quickjs-0.9.0/qjsc.c: In function 'jsc_module_loader.part.0':
/opt/local/var/macports/build/_opt_local_ppcports_devel_quickjs-ng/quickjs-ng/work/quickjs-0.9.0/qjsc.c:216:46: note: format string is defined here
  216 |         snprintf(cname1, sizeof(cname1), "%s_%d", cname, suffix_num);
      |                                              ^~
In function 'find_unique_cname',
    inlined from 'jsc_module_loader.part.0' at /opt/local/var/macports/build/_opt_local_ppcports_devel_quickjs-ng/quickjs-ng/work/quickjs-0.9.0/qjsc.c:262:13:
/opt/local/var/macports/build/_opt_local_ppcports_devel_quickjs-ng/quickjs-ng/work/quickjs-0.9.0/qjsc.c:216:42: note: directive argument in the range [1, 2147483647]
  216 |         snprintf(cname1, sizeof(cname1), "%s_%d", cname, suffix_num);
      |                                          ^~~~~~~
/opt/local/var/macports/build/_opt_local_ppcports_devel_quickjs-ng/quickjs-ng/work/quickjs-0.9.0/qjsc.c:216:9: note: '__builtin_snprintf' output between 3 and 1035 bytes into a destination of size 1024
  216 |         snprintf(cname1, sizeof(cname1), "%s_%d", cname, suffix_num);
      |         ^~~~~~~~
```

It does build if I pass `-Wno-error=format-truncation=` manually to CMake.

--- a/qjsc.c
+++ b/qjsc.c
@@ -237,7 +237,7 @@ JSModuleDef *jsc_module_loader(JSContext *ctx,
         size_t buf_len;
         uint8_t *buf;
         JSValue func_val;
-        char cname[1024];
+        char cname[1000];
 
         buf = js_load_file(ctx, &buf_len, module_name);
         if (!buf) {

